controllers/dynakube/dynakube_controller.go

```diff
@@ -178,13 +184,22 @@ func (r *ReconcileDynaKube) reconcileImpl(ctx context.Context, rec *utils.Reconc
 
 	if rec.Instance.Spec.KubernetesMonitoringSpec.Enabled {
 		upd, err := kubemon.NewReconciler(
-			r.client, r.apiReader, r.scheme, dtc, rec.Log, rec.Instance, dtversion.GetImageVersion,
+			r.client, r.apiReader, r.scheme, dtc, rec.Log, rec.Instance, dtversion.GetImageVersion, r.enableUpdates,
 		).Reconcile()
 		if rec.Error(err) || rec.Update(upd, defaultUpdateInterval, "kubemon reconciled") {
 			return
 		}
 	}
 
+	if rec.Instance.Spec.RoutingSpec.Enabled {
+		upd, err := routing.NewReconciler(
+			r.client, r.apiReader, r.scheme, dtc, rec.Log, rec.Instance, dtversion.GetImageVersion, r.enableUpdates,
+		).Reconcile()
+		if rec.Error(err) || rec.Update(upd, defaultUpdateInterval, "routing reconciled") {
+			return
+		}
+	}
```

Comment:
Please add the logic for disabling the feature.
See https://github.com/Dynatrace/dynatrace-operator/pull/67/files#diff-3091c00b5905d424b93327f07c22caa5ed45fd2683ead8e292f284f11502c508 for reference

Reply:
implemented removing statefulset for routing when routing is disabled